packages/taiko-client/proposer/proposer_test.go

```diff
@@ -113,10 +113,10 @@ func (s *ProposerTestSuite) SetupTest() {
 			TxSendTimeout:             txmgr.DefaultBatcherFlagValues.TxSendTimeout,
 			TxNotInMempoolTimeout:     txmgr.DefaultBatcherFlagValues.TxNotInMempoolTimeout,
 		},
-		GasNeededForProposingBlock: 0,
-		GasNeededForProvingBlock:   0,
-		PriceFluctuationModifier:   50,
-		OffChainCosts:              big.NewInt(0),
+		CheckProfitability:       true,
+		GasNeededForProvingBlock: 0,
+		PriceFluctuationModifier: 50,
+		OffChainCosts:            big.NewInt(0),
```

Comment:
I think all these values should be left on defualt unless you are explicitly testing them